packages/network-of-terms-reconciliation/src/preview.ts

```diff
@@ -31,6 +36,8 @@ export function preview(lookupResult: LookupQueryResult, locale: locale) {
         ${relatedTerms(locale.broader, term.broaderTerms)}
         ${relatedTerms(locale.narrower, term.narrowerTerms)}
         ${relatedTerms(locale.related, term.relatedTerms)}
+        <dt>${locale.source}</dt>
+        <dd>${source.name} (${source.creators[0]?.alternateName})</dd>
```

Comment:
I decided it would be best to start the card with the term itself rather than its source. Does it make sense to show the term source like this, at the bottom of the card, in the definition list?

Reply:
Perfect!